tests/integration/conftest.py

```diff
@@ -15,7 +22,7 @@ def capabilities(capabilities):
 @pytest.fixture
 def firefox_options(firefox_options):
     """Set Firefox Options."""
-    firefox_options.headless = True
+    firefox_options.headless = not bool(os.environ.get("DISABLE_HEADLESS"))
```

Comment:
We can get rid of this line and just use the env variable ```MOZ_HEADLESS```.

Reply:
I couldn't get it to work quickly so I filed it as a followup:

https://github.com/mozilla/experimenter/issues/2389